Main.pas

```diff
@@ -1376,6 +1378,36 @@ procedure TMainForm.WipeBoxes;
 end;
                                    
 
+procedure TMainForm.FirstTime1Click(Sender: TObject);
+const
+    Msg='First Time?'#13 +
+        'Welcome! This is the first time setup:'#13 +
+        ''#13 +
+        '1) Select the Contest you wish to operate.'#13 +
+        '2) Type the exchange you wish to send.'#13 +
+        '3) In the station section replace VE3NEA with your call.'#13 +
+        '4) Select your CW Speed, Tone, and Bandwidth.'#13 +
+        '5) Set Min/Max CW Receive speed using Settings menu.'#13 +
+        '6) Under band conditions you can select any realistic hardships.'#13 +
+        '7) Activity is the average amount of responses you want per CQ.'#13 +
+        '    So if no one responds, you might get twice the number the'#13 +
+        '    following time. This is a pile up trainer after all - have fun.'#13 +
+        '8) Select the time limit you want to run the contest.'#13 +
+        '9) The Run button has a drop down.'#13 +
+        '    - Single Call - immediately you will get someone trying to'#13 +
```

Comment:
Minor comment: reverse order to match the pulldown list.  Minor wording suggestions.
```suggestion
        '    - Pile up - Hit Enter or F1 to call CQ to start. Get ready for pileups!'#13 
        '    - Single Calls - Work one station at a time with no pileups.'#13 +
```